packages/tabs/README.md

```diff
@@ -0,0 +1,19 @@
+
+# tabs
+
+React tabs component
+
+```sh
+npm i pcln-tabs
+```
+
+```jsx
+import Tabs from 'pcln-tabs'
+```
+
+```jsx
+
```

Comment:
It'd be helpful to get an example in here first, so that people can comment on the public API and not worry about the implementation details at first